app/pods/components/web-components/au-textarea/component.js

```diff
@@ -9,4 +9,11 @@ export default class Textarea extends Component {
     }
     return DEFAULT_ROWS;
   }
+
+  get resize() {
```

Comment:
iIk zou dees persoonlijk getClass of zo noemen, kan ook voor andere doelen gebruikt worden als resize, maar is zeker goed zo.

Reply:
fixed